docs_src/microservices/core/data/Ch-CoreData.md

```diff
@@ -77,3 +77,26 @@ The two following High Level Interaction Diagrams show:
 **Core Data Request Event Reading or Data for a Device**
 
 ![image](EdgeX_CoreDataEventReading.png)
+
+## Configuration Properties
+
+Please refer to the general Configuration documentation for configuration properties common across all services.
```

Comment:
Make `general Configuration documentation` a link back to the general Configuration documentation?
Here and in all to other service config sections.